srcpkgs/bmap-tools/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'bmap-tools'
+pkgname=bmap-tools
+version=3.5
+revision=1
+archs=noarch
+build_style=python3-module
+make_install_args="--optimize=1"
```

Comment:
--optimize is not necessary. Xbps don't use cache created duiring build, it creates one on install, and it is optimized :)

Reply:
Good to know, thanks!